### ISSUE TITLE
Ping GBIF to reinterpret EML for correct author order

### ISSUE THREAD

**Issue by peterdesmet** [inbo/data-publication#129]
See [this issue](http://dev.gbif.org/issues/browse/POR-523?focusedCommentId=22515&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22515). GBIF will now show the authors of a dataset in the correct order, but it needs to reinterpret the EML of dataset to do so.

We should contact the GBIF helpdesk to do this for all INBO datasets.

**Comment by stijnvanhoey** [comment]
Mail sent to gbif helpdesk.

**Comment by peterdesmet** [comment]
Did you get any response?

**Comment by stijnvanhoey** [comment]
> All the datasets have been re-crawled. Can you please verify that the EML files were updated to your satisfaction.

**Comment by peterdesmet** [comment]
No, Peter's comment at http://dev.gbif.org/issues/browse/POR-523 is still unresolved: Koen Devos should be the first person for ORIGINATOR, METADATA AUTHOR and ADMINISTRATIVE CONTACT in http://www.gbif.org/dataset/7f9eb622-c036-44c6-8be9-5793eaa1fa1e. That is not the case.

**Comment by stijnvanhoey** [comment]
GBIF helpdesk provided the following response:

> We have fixed the issue on our side. This is a legacy bug and it will go away with the new portal release.

**Comment by peterdesmet** [comment]
Nice, let's close when it is launched.
